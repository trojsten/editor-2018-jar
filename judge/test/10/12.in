INT:
velkostmapy
20
INT:
dlzka
0
INT:
vyska
0
INT:
okuliare
0
VECTOR_STR:
mapa
20
Y;((FFI@.?V)5.PT=<##
4Y@IE7NGBURER@H?<PCW
RC6KQQYHV0Q=A-$PHJ(D
GBD7?Y;9<Z8>0YJ9XW=-
+P(5R/)XXNE'6E4XGO*@
E/BX&15K:G/CSX:[<HQ?
,U>?>9X82UFJFJ8C@G4A
W%3D3;:<+YZN;2)2MIB4
-C2)Y/6-:*FPZ[HLF?18
O6@BS*67SWQH8)Z'7T0O
KAJVBW>L@'M0J--&9,LM
6Z;D8A@,TDVOI[8?O=V>
QTE1T1=TJTR0$'P6=%ET
I72S+*/LSV92H:@?@[U<
)3P5OJ.Q&>D8'8N==P71
SD?LMAQ?'S$,BA)R:D7=
()B,YBM2MYA&>B9$<ZRP
SL%O9FEP7T8[8$46).AQ
3/D&2KSOO3BX&'=>GTJ'
F02,J*AURH(6H<AAW5HL
