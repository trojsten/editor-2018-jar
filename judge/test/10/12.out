{"dlzka": 2, "vyska": 1}